content/en/core/releases/4.10.0.md

```diff
@@ -0,0 +1,118 @@
+
+##### Details
+- The list of places will be shown alphabetically
+- The first place in the list will be selected by default
+- Only one place can be selected at a time (it’s not a multi-select)
+- The label / heading for the list of places in the filter sidebar is the [name_key](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/hierarchy/#app_settingsjson-contact_types) of the place `contact type` as set in [app_settings](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/)
+- This filter is shown for all users who have access to aggregate targets and are configured with multiple places, you don’t need to do anything to enable it (and there is no way to disable it)
+
+{{< figure src="../images/4.10.0-aggregate-targets-2.png">}}
+
+## And more...
+
+### Features
+
+- [#9194](https://github.com/medic/cht-core/issues/9194): API Endpoint for getting place by UUID
+- [#9227](https://github.com/medic/cht-core/issues/9227): Include xpath function for validation of Luhn identifiers
+
+### Improvements
+
+- [#8461](https://github.com/medic/cht-core/issues/8461): Client side error log under contacts
+- [#9231](https://github.com/medic/cht-core/issues/9231): Allow users to view aggregate targets and filter by facility_id and reporting period
+
+### Security fixes
+
+None.
+
+### Performance improvements
+
+None.
+
+### Bug fixes
+
+- [#9156](https://github.com/medic/cht-core/issues/9156): Submitter name is duplicated on the Reports Page | Detail View
+- [#9212](https://github.com/medic/cht-core/issues/9212): The "View all" filter of the History section on the Contact Page | Detail View is not working correctly
+- [#9226](https://github.com/medic/cht-core/issues/9226): Telemetry record for task:refresh isn't resetting the start time
+- [#9229](https://github.com/medic/cht-core/issues/9229): Old action bar should prevent users with multiple facilities assigned from editing the homeplace
+- [#9243](https://github.com/medic/cht-core/issues/9243): CHT Docker Helper for 4.x fails to install the TLS certificate
+- [#9264](https://github.com/medic/cht-core/issues/9264): Contact detail's telemetry is not recording the contact's type in default config. 
+- [#9265](https://github.com/medic/cht-core/issues/9265): Contact Details load facilities when user has one home place assigned
+- [#9275](https://github.com/medic/cht-core/issues/9275): The text on the Analytics Filter dropdown does not update
+- [#9284](https://github.com/medic/cht-core/issues/9284): CouchDb restart causes all services to go down
+- [#9298](https://github.com/medic/cht-core/issues/9298): CouchDb error on indexing telemetry documents
+
+### Technical improvements
+
+- [#8699](https://github.com/medic/cht-core/issues/8699): Clean and organize e2e tests: folder /contacts
+- [#9081](https://github.com/medic/cht-core/issues/9081): Add @medic/environment shared library
```

Comment:
I think this will end up a link to a URL that doesn't exist.

```suggestion
- [#9081](https://github.com/medic/cht-core/issues/9081): Add environment shared library
```